### DISCUSSION TITLE
对于Light对象的封装与支持

### DISCUSSION THREAD

**Discussion by Holy-YxY** [Acmen-Team/Epoch#5] [Ideas]
大家可以在这里讨论相关具体实现供大家作为参考实现，等到这里讨论出大概想作代码提交人便提交即可。

**Comment by Holy-YxY** [comment]
> 大家可以在这里讨论相关具体实现供大家作为参考实现，等到这里讨论出大概想作代码提交人便提交即可。

相关资料请大家查看README中文版：[README-CN](https://github.com/Acmen-Team/Epoch/blob/main/README-CN.md)

**Reply by Mostro-Complexity** [reply]
你好，可以表达一下具体的封装要求吗？我查看过代码后，感觉即便我知道如何实现，也不知道该如何开始。😂

**Reply by Holy-YxY** [reply]
> 你好，可以表达一下具体的封装要求吗？我查看过代码后，感觉即便我知道如何实现，也不知道该如何开始。😂

感谢反馈，我下来尽快列出一个详细的介绍和说明，并大体给出代码模板以及相关变动范围和需求。
